Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM quay.io/ukhomeofficedigital/alpine:v3.13
 
 ENV USER user_hocs_audit
```

Comment:
Do we still need this?

Reply:
https://wiki.alpinelinux.org/wiki/Setting_up_a_new_user - doesn't seem like you can create a user with just a UID and no name

Reply:
Yep my bad 🙂